packages/actionmenu/src/react/menu.js

```diff
@@ -115,7 +115,7 @@ const ActionMenu = React.forwardRef((props, forwardedRef) => {
       (direction === 'up' && activeIndex === 0) ||
       (direction === 'down' && activeIndex === lastIndex)
 
-    if (atEdge) navigateOut(evt)
+    if (atEdge) props.onClose(evt)
```

Comment:
Why wasn't navigateOut working before?  (It calls props.onClose internally.)

Reply:
Actually it was. Close #976